lampod/src/ln/inventory_manager.rs

```diff
@@ -41,12 +41,21 @@ impl InventoryHandler for LampoInventoryManager {
                     .map(|alias| alias.to_string());
                 // we have to put "" in case of alias missing as cln provide us with a random alias.
                 let alias = alias.unwrap_or_else(|| "".to_string());
+                let (_, height) = self.channel_manager.onchain.backend.get_best_block()?;
+                let blockheight;
+                if height.is_none() {
+                    blockheight = "Still loading...".to_string();
+                } else {
+                    // Safe to unwrap here
+                    blockheight = height.unwrap().to_string();
```

Comment:
no this should be a integer

Reply:
Then what should we put up when the blocks are still loading?

Reply:
Please see https://github.com/vincenzopalazzo/lampo.rs/pull/203#discussion_r1553820430

In addition, you can check the cln version, the syncing part is in another json field.

P.S: it is not loading, we are not a video app. We are syncing